src/services/qrService.ts

```diff
@@ -248,14 +248,19 @@ const connectOpenId4VcIssuance = async (args: IQrDataArgs): Promise<void> => {
               },
             },
           ],
-          onCreate: () => sendResponseOrSelectCredentials(metadata.credentialsSupported),
+          onCreate: () => onCreateSuccess(metadata),
         });
       } else {
         sendResponseOrSelectCredentials(metadata.credentialsSupported);
       }
     });
   };
 
+  const onCreateSuccess = async (metadata: IServerMetadataAndCryptoMatchingResponse): Promise<void> => {
+    args.navigation.goBack();
```

Comment:
technically this is not the solution that was discussed. We discussed to remove the screen from the stack. this navigates back and show the qr scanner again. which is also not the flow maria designed.

look in to calling something like

            args.navigation.reset({
              index: 0,
              routes: [{name: ScreenRoutesEnum.CREDENTIAL_DETAILS}],
            });

when we go to the credential details to reset the stack. this sets the CREDENTIAL_DETAILS as the first screen in the stack so then i would be impossible to naviugate back to the contact add screen.

only issue i ran into is that the navigator lost where it is and the goback is not handled anymore.

Reply:
args.navigation.reset({
  index: 0,  routes: [{name: ScreenRoutesEnum.QR_READER}, {name: ScreenRoutesEnum.CREDENTIAL_DETAILS}],});

will fix the goback issue

Reply:
Okay, apparently @hrehman-sphereon tried that, but didn't work. I agree that we discussed something else, but since we are already awaiting the contact creation before being able to go back on the next screen, there is no possibility for a race condition, so current approach works and I am fine with that.

Reply:
Okay just had a demo and the app is not behaving as you would expect. It quickly shows the QR code reader again, because of the go back in code, which of course is confusing.